integration_tests/suite/test_call_log_generation.py

```diff
@@ -1734,26 +1736,26 @@ def test_adhoc_conference(self):
         '''\
         linkedid      | uniqueid      | eventtime                | eventtype                 | channame                | cid_name   | cid_num   | extra
         --------------+---------------+--------------------------+---------------------------+-------------------------+------------+-----------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
-        1674248467.40 | 1674248467.40 | 2023-01-20 00:00:00.0+00 | CHAN_START                | PJSIP/rgcZLNGE-00000028 | test-UC-1  | 8000      |
-        1674248467.40 | 1674248467.40 | 2023-01-20 00:00:01.0+00 | XIVO_INCALL               | PJSIP/rgcZLNGE-00000028 | test-UC-1  | 8000      | {"extra":"54eb71f8-1f4b-4ae4-8730-638062fbe521"}
-        1674248467.40 | 1674248467.40 | 2023-01-20 00:00:02.0+00 | WAZO_CALL_LOG_DESTINATION | PJSIP/rgcZLNGE-00000028 | test-UC-1  | 8000      | {"extra":"type: user,uuid: ad5b78cf-6e15-45c7-9ef3-bec36e07e8d6,name: A Mctest"}
-        1674248467.40 | 1674248467.40 | 2023-01-20 00:00:03.0+00 | APP_START                 | PJSIP/rgcZLNGE-00000028 | test-UC-1  | 8000      |
+        1674248467.40 | 1674248467.40 | 2023-01-20 00:00:00.0+00 | CHAN_START                | PJSIP/rgcZLNGE-00000028 | Operator  | 8000      |
```

Comment:
Why did this user get renamed to Operator? This scenario is not specific to the switchboard. Naming the user Operator is misleading

Reply:
Ooof, that's a find/replace messup I should have caught before committing